terraform/runner/main.tf

```diff
@@ -0,0 +1,58 @@
+locals {
+
+}
+
+module "object_store_instance" {
+  source = "github.com/GSA-TTS/terraform-cloudgov//s3?ref=v1.0.0"
+
+  cf_org_name      = var.cf_org_name
+  cf_space_name    = var.cf_space_name
+  name             = var.object_store_instance
+}
+
+resource "cloudfoundry_app" "gitlab-runner" {
+    name = var.runner_name
+    space = data.cloudfoundry_space.space.id
+    path = "${path.module}/files/src.zip"
+    buildpacks = ["https://github.com/cloudfoundry/apt-buildpack", "binary_buildpack"]
+    instances = 1
+    command = "gitlab-runner run"
+    memory = var.runner_memory
+    health_check_type = "process"
+    
+    environment = {
```

Comment:
thought (non-blocking): Maintaining this manifest separately from `/manifest.yml` will be toilsome as we go. Might be worth a later PR to unify this in some way. (*Said way, not defined!)